README.md

```diff
@@ -6,10 +6,10 @@
 
 ## Purpose
 
-This is a parser for the [Web IDL](https://heycam.github.io/webidl/) language. If
-you don't know what that is, then you probably don't need it. It is meant to be used
-both in Node and in the browser (the parser likely works in other JS environments, but
-not the test suite).
+This is a parser for the [Web IDL](https://heycam.github.io/webidl/) language that
+[powers web specifications](https://heycam.github.io/webidl/#introduction). It is meant
+to be used both in Node and in the browser (the parser likely works in other JS
+environments, but not the test suite).
```

Comment:
Maybe:

This is a parser for the [Web IDL](https://heycam.github.io/webidl/) language. Web IDL is a format used by web-related documents and tools to formally specify the APIs which are implemented by browsers. You can learn more about the language by reading [its specification](https://heycam.github.io/webidl/#introduction).

To my ear, “powers web specifications” implies that WebIDL is an executable language of some sort.

Up to you though! Thanks for asking.

Reply:
Thanks! Tweaked it a bit more to describe what the language is for. The introduction link should be enough for further description.